src/issue/screens/issue.screen.js

```diff
@@ -40,7 +40,8 @@ const mapStateToProps = state => ({
 });
 
 const mapDispatchToProps = dispatch => ({
-  getIssueCommentsByDispatch: url => dispatch(getIssueComments(url)),
+  getIssueCommentsByDispatch: issueCommentsURL =>
+    dispatch(getIssueComments(issueCommentsURL)),
```

Comment:
If I'm not mistaken `prettier` does this with longer lines but I could be wrong. I'm fine with it either way